### PR TITLE
Fixed Spellings

### PR DESCRIPTION
**Pull request by Sagar-Wadhwa-726**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,28 @@
 # Kubecon-India-2024-sponsorship
+
 ![Image](https://github.com/user-attachments/assets/e1e03137-31c6-4322-bee6-d4d8e7f4be57)
 
-* Hello folks 👋
-* We are glad to announce that we are running this programme to sponsorship the KubeCon-India-2024 tickets.
---------
+- Hello folks 👋
+- We are glad to announce that we are running this programme to sponsorship the KubeCon-India-2024 tickets.
```
Comment (SaumyaBhushan) [inline README.md:6]: Can we change sponsorship to sponsor here ?

File: README.md [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
+
+---
 
 ## Criterias:
+
 1. You have to write a blog on topic "How GrowInCommunity helped you in rise the carreer in tech".
 2. After writting the blog, you have to post it on your self own domain (that can be free domain or purchased one).
```
Comment (SaumyaBhushan) [inline README.md:13]: writting - > writing ?
self own - > self-owned ?

File: README.md [hunk 3]
```diff
@@ -1,23 +1,28 @@
...
-* We are glad to announce that we are running this programme to sponsorship the KubeCon-India-2024 tickets.
---------
+- Hello folks 👋
+- We are glad to announce that we are running this programme to sponsorship the KubeCon-India-2024 tickets.
+
+---
 
 ## Criterias:
+
 1. You have to write a blog on topic "How GrowInCommunity helped you in rise the carreer in tech".
```
Comment (SaumyaBhushan) [inline README.md:12]: A better title would be "How GrowInCommunity helped you advance your career in tech ?"

File: README.md [hunk 4]
```diff
@@ -1,23 +1,28 @@
...
 2. After writting the blog, you have to post it on your self own domain (that can be free domain or purchased one).
 3. Share the blog on socials: Twitter/X and LinkedIn with hashtag #growincommunity #kubeconindia2024
-4. We have a dedicated chaanel om our [Discord Server](https://discord.gg/SXqTuNBm4Z), where yoi can communicate wity the team if you have any doubt or need help.
+4. We have a dedicated channel on our [Discord Server](https://discord.gg/SXqTuNBm4Z), where you can communicate with the team if you have any doubt or need help.
 5. You also have to share the bolg link in our discord server channel to verified by the team.
```
Comment (SaumyaBhushan) [inline README.md:16]: bolg - > blog .
to verified - . to be verified ?

**Review by SaumyaBhushan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Akshaykumar05) [reply README.md:6]: Yes sure

File: README.md (hunk 3 above)
Comment (Akshaykumar05) [reply README.md:12]: Yeah, that looks much better.

File: README.md (hunk 4 above)
Comment (Akshaykumar05) [reply README.md:16]: 👍

**Comment by Akshaykumar05** [comment]
All looks good to me. Thanks for the contribution!

**Pull request merged**